packages/neuron-ui/src/components/PageContainer/index.tsx

```diff
@@ -148,7 +152,7 @@ const PageContainer: React.FC<ComponentProps> = props => {
   )
 }
 
-export const Breadcrumbs: FC<PropsWithChildren<{}>> = ({ children }) => {
+export const Breadcrumbs: FC<PropsWithChildren<object>> = ({ children }) => {
```

Comment:
We only use `children`, why should we add `object` for `PropsWithChildren`?

Reply:
I guess you should change to `Record<string, never>`, because you want a type meaning "empty object"

Reply:
It's my mistake, it will throw an exception when using `Record<string, never>` because `type PropsWithChildren<P> = P & { children?: ReactNode | undefined };`, you can replace `object` to `React.ReactNode`